requirements-60.txt

```diff
@@ -0,0 +1,12 @@
+# from https://dist.plone.org/release/6.0.0a4/requirements.txt
```

Comment:
@mauritsvanrees if I recall correctly we agreed that we will consider all files on dist.plone.org as release artifacts (except "latest" of course) which means they must not change under any circumstances. If we agree on this we could also rely on dist.plone.org directly. If we can not make that commitment, we should keep a copy IMO.

Reply:
I treat them as unchangeable. In 6.0.0a4 after release a problem with `plone.volto` was discovered and then fixed. Since this was within a day (half a day even) and in an alpha release, I decided to change the version pin. That is definitely an exception.

Reply:
If we treat dist.plone.org as a host for release artifacts there should be no exception from that rule IMHO. In that case, we should make a new release. If a CI system can not 100% rely on an external resource to not be changed it shouldn't use them. That's a lesson I had to learn the hard way over and over again in the past years.